dependency-check-maven-config.xml

```diff
@@ -0,0 +1,485 @@
+        is to prevent OWASP from complaining.
+
+        file name: apache-karaf-4.0.4.zip: org.apache.servicemix.bundles.not-yet-commons-ssl-0.3.11_1.jar
+        ]]></notes>
+        <cve>CVE-2004-0009</cve>
+    </suppress>
+
+
+    <suppress>
+        <notes><![CDATA[
+                CVE-2015-5344 is an issue with Camel version before 2.16.1
+        OWASP appears to have confused the internal proxy-camel-servlet
+        version with the overall Camel version - marking as false positive.
+
+        file name: proxy-camel-servlet-2.10.0-SNAPSHOT.jar]]></notes>
+        <cve>CVE-2015-5344</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+            CVE-2014-1939 is an Android specific issue, and does not apply here
+            file name: google-http-client-1.22.0.jar]]></notes>
+        <cve>CVE-2014-1939</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2008-0660 is a stack based buffer overflow vulnerability related to ActiveX and
+            several image uploaders. This is unrelated to presto-parser, so marking as a false
+            positive.
+        </notes>
+        <cve>CVE-2008-0660</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2007-1085 is a cross-site scripting (XSS) vulnerability related to Google Desktop
+            which does not apply here.
+        </notes>
+        <cve>CVE-2007-1085</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2007-3150 is a JavaScript injection vulnerability related to Google Desktop which
+            does
+            not apply here.
+        </notes>
+        <cve>CVE-2007-3150</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2010-1807 is related to a client side/browser vulnerability in WebKit. Marking the
+            vulnerability as a false positive since the vulnerable code is not currently used and
+            the code is executed server-side.
+        </notes>
+        <cve>CVE-2010-1807</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2011-2730 is related to a vulnerability in the VMware SpringSource Spring
+            Framework, where OWASP flags jars that are unrelated or have no dependency on
+            Spring, so marking it as a false positive.
+        </notes>
+        <cve>CVE-2011-2730</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2011-5034: Applies to
+            org.apache.servicemix.specs.activation-api-1.1-2.5.0.jar/META-INF/maven/org.apache.geronimo.specs/geronimo-activation_1.1_spec/pom.xml
+            ServiceMix embeds some Specs provided by Geronimo but does not use any of the effected
+            libraries.
+        </notes>
+        <cve>CVE-2011-5034</cve>
+    </suppress>
+
+
+    <suppress>
+        <notes><![CDATA[
+                Suppressing vulnerabilities CVE-2013-4221 and CVE-2013-4221 as the offending jar file (org.restlet-2.1.1.jar)
```

Comment:
Should the second CVE-2013-4221 be CVE-2013-4271?
Should we just remove the reference to the 4271 and copy the notes there as well?
That way each suppression has documented notes.